css/index.css

```diff
@@ -1,3 +1,454 @@
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 5px;
+}
+
+h1{
+	font-size: 50px;
+}
+/* 
+	Main Page CSS
+*/
+@keyframes button-hover {
+	from {background-color: gray;
+		  color:white;}
+	to {background-color: white;
+		color: gray;}
+}
+
+@keyframes button-not-hover {
+	from {background-color: white;
+		color: gray;}
+	to {background-color: gray;
+		color:white;}
+}
+
+*{
+	box-sizing: border-box;
+}
+
+body{
+    display: block;
+	width: 100%;
+	height: 100%;
+	max-width: 700px;
+	margin: auto;
+}
+
+#header-placeholder{
+	display: inline-block;
+	width: 100%;
+	height: 50px;
+	margin-top: 10px;
+	margin-bottom: 35px;
+}
+
+header{
+	position: fixed;
+	display: flex;
+    top: 10px;
+	width: 100%;
+	max-width: 700px;
+	height: 50px;
+	background-color: darkgray;
+	z-index: 1;
+	border-radius: 10px;
+	align-items: center;
+}
+
+header nav{
+	width: 570px;
+}
+
+header nav a{
+	padding-right: 17px;
+	padding-left: 17px;
+	text-decoration: none;
+	color: black;
+	font-size: 14px;
+	background-color: gray;
+	border-radius: 5px;
+	margin: 5px 5px 5px 5px;
+}
+
+header nav a:hover{
+ 	animation-name: button-hover;
+  	animation-duration: .5s;
+  	animation-fill-mode: forwards;
+}
+
+header nav a:not(:hover){
+	animation-name: button-not-hover;
+	animation-duration: .5s;
+	animation-fill-mode: forwards;
+}
+
+.logo{
+	width: 120px;
+	height: 20px; 
+	background-image: url("../img/logo.png");
+	background-size: 100% 100%; 
+}
+
+.header-double{
+	vertical-align: top;
+	display: flex;
+	width: 100%;
+	margin: 15px 0px 50px 0px;
+	font-size: 14px;
+}
+
+.header-double section{
+	max-width: 350px;
+	width: 100%;
+}
+
+#header-img{
+	width: 300px;
+	height: 300px;	
+	background-image: url("../img/header-img.png");
+	background-size:100% 100%;
+}
+
+.header-double button{
+	display: block;
+	margin: auto;
+	margin-top:15px;
+	width: 125px;
+	height: 25px;
+	background-color: white;
+	border-color: black;
+	border-width: 1px;
+	cursor: pointer;
+}
+
+#header-text{
+	padding-top: 45px;
+}
+
+.header-double h1{
+	text-align: center;
+	width: 100%;
+}
+
+hr{
+	border-width: .5px;
+	border-color: black;
+	width: 100%;
+}
+
+.double{
+	display: flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	width: 100%;
+	font-size: 14px;
+}
+
+.double section{
+	margin: 20px 5px 20px 5px;
+	padding: 0px;
+	width: 45%;
+}
+
+h2{
+	font-size: 18px;
+	font-weight: 200%;
+}
+
+.middle-img{
+	width: 100%;
+	height: 12.5%;
+	background-image: url("../img/mid-page-accent.jpg");
+	background-size: 100% 100%;
+}
+
+.triple{
+	display: flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	width: 100%;
+	font-size: 14px;
+}
+
+.triple section{
```

Comment:
These class names are fine for the time being but eventually you'll want to be more specific. For example, instead of .triple something more like .second-main-content-section. it's alot longer but also much clearer about what this css is effecting which makes it easier for others to read yo code.